spec/spec_helper.rb

```diff
@@ -11,6 +11,13 @@ class Undef
   def inspect; 'undef'; end
 end
 
+# Workaround for slow rspec-puppet-facts
+def supported_os_opts
```

Comment:
Or even `on_supported_os_opts` as it's the options hash for the `on_supported_os` method.

This should not be a permanent workaround, the underlying issue should be fixed somehow.